src/index.html

```diff
@@ -11,7 +11,134 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form  action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="fieldset_distance fieldset__first">
+        <legend>
+          Personal information
+        </legend>
+        <div class="form-field input input__first">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          id="surname"
+          required>
```

Comment:
Closing bracket should be on the same level as opening one. Fix it and indentations (should be 2 spaces) through all your document.
```suggestion
          <input
		    type="text"
		    id="surname"
		    required
          >
```